src/redux/actionTypes/genes.js

```diff
@@ -60,9 +60,11 @@ const MARKER_GENES_LOADED = `${GENES}/markerGenesLoaded`;
  */
 const MARKER_GENES_ERROR = `${GENES}/markerGenesError`;
 
+const GENE_ORDER_SET = `${GENES}/geneOrderSet`;
```

Comment:
as I suggested during our call, will be best to keep this elsewhere, since it is something specifically applicable to the marker heatmap in plots and tables, it is not used anywhere else.